src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue19127.cs

```diff
@@ -23,8 +24,8 @@ public void ContentOfFrameShouldChange()
 
 			var textAfterClick = App.FindElement("label2").GetText();
 
-			Assert.AreEqual(textBeforeClick, "Camera is Disabled");
-			Assert.AreEqual(textAfterClick, "Camera is Enabled");
+			Assert.That(textBeforeClick!, Is.EqualTo("Camera is Disabled"));
```

Comment:
Failing on Android and Catalyst:
<img width="294" alt="image" src="https://github.com/user-attachments/assets/af3c2bf6-b71d-4f94-a68c-537b929c27d0">

Reply:
Can wy try again?